package.json

```diff
@@ -4,7 +4,8 @@
   "private": true,
   "scripts": {
     "start": "node node_modules/react-native/local-cli/cli.js start",
-    "test": "jest"
+    "test": "jest",
+    "release-package" : "cd usabilla-react-native && npm run semantic-release"
```

Comment:
I would just keep these command names short, why not use `release`?

Reply:
I wanted to avoid the confusion between the `release-package` command in the DevApp and the actual `release` command in the `usabilla-react-native`

Reply:
I will change it to `release`